### PR TITLE
Add support for Node.js 20

### PR DESCRIPTION
**Pull request by JulienItard**
Hello !

see https://github.com/achrinza/node-ipc/issues/45

### CODE REVIEW

**Review by jokay — APPROVED** [review]

**Comment by JulienItard** [comment]
@jokay thanks for approval but i see that workflow is waiting for approval ?

**Comment by jokay** [comment]
Yes, no idea why the maintainer does not merge and release it :shrug:

**Comment by adamJLev** [comment]
@achrinza hi help. Check this out when you can please

File: package.json
Comment (jokay) [inline package.json:19]: What about this? @achrinza :wink:


```suggestion
    "node": ">=14"
```

See [documentation](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines).

**Pull request closed without merging**
